studips/src/donneedur.txt

```diff
@@ -0,0 +1,132 @@
+const users = [
```

Comment:
data are in json: why not using a .json extension?

please use english for file names: data.json

Reply:
donneedur.txt has been renamed as "data.json". It has also been reformated and moved into the "database" folder. Is it the right place though ?